src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementSpecHandler.java

```diff
@@ -0,0 +1,330 @@
+    }
+
+    private String getBody(CaseData caseData) {
+
+        return format(JUDGMENT_GRANTED, format(
+            "/cases/case-details/%s#Claim documents",
+            caseData.getCcdCaseReference()
+        ));
+        // return format(CPR_REQUIRED_INFO);
+    }
+
+    private CallbackResponse validateDefaultJudgementEligibility(CallbackParams callbackParams) {
+
+        var caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
+        ArrayList<String> errors = new ArrayList<>();
+        if (nonNull(caseData.getRespondent1ResponseDeadline())
+            && caseData.getRespondent1ResponseDeadline().isAfter(LocalDateTime.now())) {
+            String formattedDeadline = formatLocalDateTime(caseData.getRespondent1ResponseDeadline(), DATE_TIME_AT);
+            errors.add(format(NOT_VALID_DJ, formattedDeadline));
+        }
+        List<String> listData = new ArrayList<>();
+        listData.add(caseData.getRespondent1().getIndividualFirstName()
+                         + " "
+                         + caseData.getRespondent1().getIndividualLastName());
+        if (nonNull(caseData.getRespondent2())) {
+            listData.add(caseData.getRespondent2().getIndividualFirstName()
+                             + " "
+                             + caseData.getRespondent2().getIndividualLastName());
+            listData.add("Both Defendants");
+            caseDataBuilder.defendantDetailsSpec(DynamicList.fromList(listData));
+        }
+        caseDataBuilder.defendantDetailsSpec(DynamicList.fromList(listData));
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .errors(errors)
+            .data(errors.size() == 0
+                      ? caseDataBuilder.build().toMap(objectMapper) : null)
+            .build();
+    }
+
+    private CallbackResponse checkStatus(CallbackParams callbackParams) {
+
+        var caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(caseDataBuilder.build().toMap(objectMapper))
+            .build();
+    }
+
+    private CallbackResponse partialPayment(CallbackParams callbackParams) {
+
+        var caseData = callbackParams.getCaseData();
+
+        var totalIncludeInterest = caseData.getTotalClaimAmount().doubleValue()
+            + caseData.getTotalInterest().doubleValue();
+        List<String> errors = new ArrayList<>();
+
+        if (caseData.getPartialPayment() == YesOrNo.YES) {
+            var partialPaymentPennies = new BigDecimal(caseData.getPartialPaymentAmount());
+            var partialPaymentPounds = MonetaryConversions.penniesToPounds(partialPaymentPennies).doubleValue();
+            if (partialPaymentPounds >= totalIncludeInterest) {
+                errors.add("The amount already paid exceeds the full claim amount");
+            }
+        }
+
```

Comment:
this could give checkstyle error (too many blank lines in a method)

Reply:
I believe this is already in master styled like this?